### ISSUE TITLE
Architecture 1 Detection Model + Regression Model training, testing and evaluation.

### ISSUE THREAD

**Issue by grewe** [grewe/SideWalk#1]
Training using 2 collabs one to create Damage detention model (1 class) and the second to create Regression Analysis (on detected sub-images).   

Evaluation: For both, evaluate with Tensorboard graphs.  For Detection evaluation with Coco Metrics.  For Regression evaluate with MAE (Mean Absolute Error).

Testing:  Run each model with test data and store results in drive.

Write up results in the Capstone Report for training, testing and evaluation.

**Comment by grewe** [comment]
NOTE: [ if you have multiple training runs and hence multiple results then you need to make appropriately named sub-directories and a README file in the main (i.e. Code/results/training) directory that says what each run is for.  For example:

- run1  (for epochs 300, batch 16, ......parameters)  make Code/Detection/results/run1/training and Detection/results/run2/testing
- run2 ( for epochs 300, batch 32, learning rate 0.002......) make Code/Detection/results/run2/training and Detection/results/run2/testing

and similar for REGRESSSION..

DOCUMENT All results (for each run) **must be documented in the capstone**  

(below is showing for case of only 1 run)
FOR DETECTION:   

- README = documents for (each ) training run the parameters (learning rate, epocs, e.....)
- Colab copy of it AND a shortcut to Code/Dectection
- any files or screenshots , screendump (to text file of training epoch info printed to screen) etc. saved in Code/Detection/(runX)/results/training
- put the Tensorboard log files into Code/Detection/(runX)/results/training/logs
- put the Tensorfboard MODEL files including TFLITE (BOTH) in Code/Detection/(runX)/results/training/model
- any output of testing results goes to Code/Detection/(runX)/results/testing  [Coco Metrics  + actual results on testing data] 

FOR REGRESSION: 

-  Colab copy of it AND a shortcut to Code/Regression
- any files or screenshots , screendump (to text file of training epoch info printed to screen) etc. saved in Code/Regression/(runX)/results/training
- put the Tensorboard log files into Code/Detection/(runX)/results/training/logs
- put the Tensorfboard MODEL files including TFLITE (BOTH) in Code/Detection/(runX)/results/training/model
-  any output of testing results goes to Code/Regression/results/testing [MAE + acttual results on testing data]
